src/content/configuration/index.mdx

```diff
@@ -45,1128 +46,46 @@ If for some reason you want to use a different configuration file depending on c
 
 ## Options
```

Comment:
We may want to change this heading now since we don't show options now. Maybe something like `Scaffolding a webpack project`